crucible/src/Lang/Crucible/CFG/SSAConversion.hs

```diff
@@ -126,24 +131,24 @@ data BlockInput ext s blocks ret args
              -- | Arguments expected by block.
            , binputArgs       :: !(Assignment (Value s) args)
            , binputStmts      :: !(Seq (Posd (Stmt ext s)))
-           , binputTerm       :: !(Posd (TermStmt s ret))
+           , binputTerm       :: !(Posd (Either (JumpInfo s blocks) (TermStmt s ret)))
```

Comment:
I think we should probably have a fresh datatype here with more descriptive constructor names rather than just using `Either`